src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --header-size: 60px;
+}
+
 body {
+  padding: 0;
```

Comment:
Don't use * selector (it impacts performance). Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for browser.